packages/runtime/runtime-definitions/src/summary.ts

```diff
@@ -123,10 +123,32 @@ export interface IGarbageCollectionSnapshotData {
 	deletedNodes: string[] | undefined;
 }
 
+/**
+ * @experimental - Can be deleted/changed at any time
+ * Contains the necessary information to allow DDSes to do incremental summaries
+ */
+export interface IIncrementalSummaryContext {
```

Comment:
Add experimental to the interface name as well to make it clear that this is experimental and no one should add any dependency to it.